fastseq_cli/transformers_generate.py

```diff
@@ -354,7 +368,12 @@ def run_generate():
                         default=None, required=False)
     parser.add_argument("--max_gen_length", type=int,
                         help="max length for generation",
-                        default=None, required=False)
+                        default=None, required=False),
+    parser.add_argument("--max_new_tokens", type=int,
```

Comment:
If `max_gen_length` and `max_new_tokens` serve the same purpose, could we just use one of them?

Reply:
both exist already in hf. This change just makes the existing functionality available in fastseq.  https://github.com/huggingface/transformers/blob/3f76bf54ff3754f9f6639ee8f9c36775be9b8ce6/src/transformers/generation_utils.py#L829

Reply:
Gotcha.